Cargo.toml

```diff
@@ -83,7 +83,7 @@ parking_lot = "0.12.3"
 testresult = "0.4.1"
 nested_enum_utils = "0.1.0"
 iroh-io = "0.6.1"
-testdir = "0.9.1"
+testdir = "0.7"
```

Comment:
Just a wild shot in the dark to get the MSRV test to pass.

Got this in the MSRV test
```
error: package `cargo-platform v0.1.9` cannot be built because it requires rustc 1.78 or newer, while the currently active rustc version is 1.76.0
Either upgrade to rustc 1.78 or newer, or use
cargo update cargo-platform@0.1.9 --precise ver
where `ver` is the latest version of `cargo-platform` supporting rustc 1.76.0
```

and testdir seems to be the culprit
```
❯ cargo tree -i cargo-platform
cargo-platform v0.1.9
└── cargo_metadata v0.14.2
    └── testdir v0.9.1
        [dev-dependencies]
        └── iroh-docs v0.28.0 (/Users/rklaehn/projects_git/iroh-docs)
```

So I would have to do some kind of bisect to find out when testdir started using a version of cargo-platform that requires rustc 1.78. Instead I just chose a random number.

Reply:
cc @flub

Reply:
Why are we checking MSRV on dev-dependencies?